### ISSUE TITLE
Detect version of next group ID on classpath

### ISSUE THREAD

**Issue by renatocaval** [akka/akka-persistence-jdbc#338]
This is the reverse of #285. It should be added on 3.5.x and check if the next version, using artifact id `com.lightbend.akka` is on the classpath.

**Comment by WellingR** [comment]
Actually if the next group id is NOT on the classpath, we could also log a message informing users that te group id has changed

**Comment by renatocaval** [comment]
> Actually if the next group id is NOT on the classpath, we could also log a message informing users that te group id has changed

I just realised that we can't do it. We will cut 3.5.3 much before we have a 4.0.0  (or 3.6.0) release. If we give a warning, users will try to update to a version that doesn't exist yet.

**Comment by WellingR** [comment]
Can't we release 3.5.3 and 3.6.0 at the same time?

**Comment by patriknw** [comment]
I think we want people to update to 2.5.3 so that we have the check in place in the old version. Therefore I think it's better to wait a bit with 3.6.0.

**Comment by renatocaval** [comment]
@patriknw, I think @WellingR point is to add a warning in 3.5.3 telling people to move to 3.6.0. And 3.6.0 will be feature-wise identical to 3.5.3, but using groupdId `com.lightbend.akka`.

**Comment by renatocaval** [comment]
@WellingR, I got some more insight into what Patrik had in mind. 

His point is: if we only release 3.5.3 we give people time to upgrade and be exposed to the check. Otherwise, people will simply jump to 3.6.0. 

And here is what I just realised:

If you are on `com.github.dnvriend %% akka-persistence-jdbc % 3.5.2` and you update to `3.6.0` or `4.0.0`, you get an error if you don't change the groupId. So, direct users are protected anyway.

The check has only one reason to exist. If you are using some framework (eg: Lagom) that transitively pulls in the plugin and at the same time you are adding the plugin directly, you will get an error if you mix the versions. 

I don't think there are many people out there pulling the plugin transitively and adding it directly at the same time. Even Lagom users tend to not do it. 

On the other hand, it's good to have a 3.5.3 with the quick-fix for the tags (#340) and the envelop timestamp (#337). Lagom's next patch release can use it and we will have a good version for other existing users that don't want to run the `4.0.0` migration immediately and stay on `3.5.x`.
